### ISSUE TITLE
Jotter doesn't behave properly when press esc on windows

### ISSUE THREAD

**Issue by Hualin** [aziz/PlainNotes#21]
Jotter is a nice idea. But it doesn't behave properly when press esc on windows. Which, when in the jotter panel, one input something there, and press esc, the panel doesn't really hide. Then i try to click the main panel, and press esc, the jotter panel hide, but then check Notes:: Inbox, nothing there. I consider it might be the issue with key binding, that it never trigger the function `SaveJotAndHidePanelCommand`.

I use windows 7 and subline text 3 btw.

cheers

**Comment by aziz** [comment]
That's for reporting. I'll look into it as soon as I set up my Windows VM machine. hopefully this weekend.

**Comment by Hualin** [comment]
thanks looking forward your updates

**Comment by eumesmo1** [comment]
Nice work!
Unfortunately I'm also having the same problem here. ESC key doesn't work.
Thanks.

**Comment by aziz** [comment]
could you please report your full version of sublime.

**Comment by MartinMuzatko** [comment]
Same issue here on Win7. Escape does not trigger ``` SaveJotAndHidePanelCommand ```
Build 3083. (latest)

**Comment by fk128** [comment]
The issue is caused by the jotter_date_format and jotter_time_format. If you change in your user preferences for plainNotes this:
```
"jotter_date_format" :"%-d %-b %Y",
"jotter_time_format": "%-I:%M %p",
```
to
```
  "jotter_date_format" :"%d %b %Y",
  "jotter_time_format": "%I:%M %p",
```
it should work. Otherwise the problem should be fixed if you update to 0.1.4.
